probcal/utils/configs.py

```diff
@@ -117,6 +117,8 @@ def from_yaml(config_path: str | Path) -> TrainingConfig:
         dataset_type = DatasetType(config_dict["dataset"]["type"])
         if dataset_type == DatasetType.TABULAR:
             dataset_path = Path(config_dict["dataset"]["path"])
+        elif dataset_type == DatasetType.IMAGE:
+            dataset_path = ImageDatasetName(config_dict["dataset"]["path"])
```

Comment:
Porter is making some upstream changes to sort out a better way to handle `dataset_path` when there isn't a specific file that defines the dataset like in the tabular case. Let's wait on that and then I can provide some guidance for how to handle this

Reply:
Okie dokie